tasks/install.yml

```diff
@@ -31,7 +31,3 @@
     version: "{{ app_tag }}"
     accept_hostkey: yes
     key_file: "{{ app_ssh_key_path }}"
-
-- name: Sync dependencies
-  ansible.builtin.command:
-    cmd: "{{ uv_bin }} sync --locked --directory {{ app_code_dir }}"
```

Comment:
One option would be to prepend this with `sudo -u {{ app_user }}` to run that as that user if you want to preserve the "check" that the playbook is doing for the environment.

Reply:
True. For now I just want to merge, but I might do that in the future as an additional sanity check.